### DISCUSSION TITLE
gpt-3.5-turbo

### DISCUSSION THREAD

**Discussion by ghost** [sashabaranov/go-openai#94] [General]
I try GPT3Dot5Turbo and GPT3Dot5Turbo0301 but show me  "error, status code: 404, message: This is a chat model and not supported in the v1/completions endpoint. Did you mean to use v1/chat/completions?"

pls fix

**Comment by lytum** [comment]
same issue from my side

**Comment by asimihsan** [comment]
In https://github.com/asimihsan/openai-discord-bot/blob/main/src/openai/openai.go I'm able to successfully use the v1.3.1 branch at commit 58d99eb. Thanks for the fast turnaround in adding this model.

**Reply by lytum** [reply]
thanks, now it works in my side.
